CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ All notable changes to this project will be documented in this file.
 * ### `nidcpower` (NI-DCPower)
     * #### Added
     * #### Changed
+        * Addressed [#1596](https://github.com/ni/nimi-python/issues/1596) by removing RLock replacements for session.py::_SessionBase::lock/unlock
```

Comment:
Changelog should be used only for customer-visible changes.

The use of RLock had a minor enough customer impact (Access to multiple nidcpower sessions are serialized) that we decided not to mention in the Changelog. So removal of it doesn't need to be mentioned either.